intrinsic_cal/CMakeLists.txt

```diff
@@ -95,7 +95,7 @@ set(CMAKE_CXX_FLAGS ${CMAKE_CSS_FLAGS} "-fPIC")
 ## CATKIN_DEPENDS: catkin_packages dependent projects also need
 ## DEPENDS: system dependencies of this project that dependent projects also need
 catkin_package(
-  INCLUDE_DIRS include /usr/include/eigen3
+  INCLUDE_DIRS /usr/include/eigen3
```

Comment:
Afaik every properly packaged Eigen install should install either a Find script, or a config file.

But for ROS packages, it's probably better to use `FindEigen.cmake` from `cmake_modules`. At least as long as system versions don't install one.